specs/schemes/exact/scheme_exact_algo.md

```diff
@@ -0,0 +1,173 @@
+# Scheme: `exact` on `Algorand`
+
+## Summary
+
+The `exact` scheme on Algorand uses the `Algorand Standard Asset (ASA)`, which are native assets (no contract required) on the Algorand blockchain to authorize a transfer of a specific amount from the payer to the resource server. The approach results in the facilitator having no ability to direct funds anywhere but the address specified by the resource server in `paymentRequirements`.
+
+## Payment Requirements asset definition 
+
+In the `exact` scheme on Algorand, the `paymentRequirements.asset` must be a string representing an `Algorand Standard Asset (ASA)` ID unsigned integer instead of an `ERC20` contract address. This **MUST** be validated by the resource server to ensure the `asset` field is valid when using the Algorand scheme.
+
+
+## `X-Payment` header payload
+
+The `payload` field of the `X-PAYMENT` header must contain `paymentGroup` as a field. It represents an atomic group of transactions as an array. This is natively supported by Algorand and does not require contracts.
+
+A group can have transactions of different types, such as `pay` (payment) and `axfer` (asset transfer). 
```

Comment:
Just a sanity check: Are we excluding other transaction types on purpose? Or should they be supported as part of the group? What if we have an asset freeze in the same group to comply with KYC requirements? Or an App Call?

Reply:
We are not excluding other types, will add a comment explaining